### DISCUSSION TITLE
quasar dev not working?

### DISCUSSION THREAD

**Discussion by faridaf97** [quasarframework/quasar#13429] [Quasar App Extensions] [answered]

**Comment by MilosPaunovic** [comment, marked as answer]
What do you mean by it not working? Do you get any error?

When you say you've just installed Quasar, do you mean the CLI, from your previous discussion thread, #13426?

The best way forward would be to take a deep dive into official documentation and their getting started guide:
[https://quasar.dev/start/quasar-cli](https://quasar.dev/start/quasar-cli)

**Reply by faridaf97** [reply]
Hi ☺️ 
Thank you very much for your patience and reply! I did use the website and youtube tutorials as a lead but  I guess my Mac Terminal didn't want the command to work. I then tried working with Visual Studio Code and it strangely worked.
